source/_patterns/01-elements/input/input.scss

```diff
@@ -1,50 +1,145 @@
 @charset "utf-8";
 
 @use "@db-ui/base/build/scss/variables" as *;
+@use "@db-ui/base/build/scss/variables.global" as *;
 @use "@db-ui/base/build/scss/helpers/functions" as *;
-@import "input.variables";
-@import "../form-elements";
+@use "@db-ui/base/build/scss/icon/icons.helpers" as *;
+
+$icons-path: "../../../icons/" !default;
+
+/* stylelint-disable */
+
+@mixin label-focus-animation($translationX: var(--elm-input-padding-left)) {
+	$transformScale: -1.25;
+
+	transition: opacity 0.08s, transform 0.15s;
+	opacity: var(--db-input-focus-opacity);
+
+	transform: translate($translationX, calc($db-sizing-md * $transformScale))
+		scale(var(--db-input-label-scale));
+}
 
 .elm-input {
-	@extend %form-element;
+	position: relative;
 
-	color: $input---color;
+	--elm-input-padding-left: #{$db-spacing-fixed-sm};
+	--elm-input-padding-right: #{$db-spacing-fixed-sm};
```

Comment:
let's start to use the new semantic names for padding and margin instead of `left` and `right`:
https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline-start

Reply:
chaned in https://github.com/db-ui/core/pull/198/commits/9291c0e751599052a5c87dc858f92b0e719c1eaa